cogboard-app/src/main/kotlin/com/cognifide/cogboard/security/LoginHandler.kt

```diff
@@ -78,6 +78,6 @@ class LoginHandler : RoutingHandlerFactory {
     }
 
     companion object {
-        private const val SESSION_DURATION_IN_SECONDS = 30 * 60
+        private const val SESSION_DURATION_IN_SECONDS = 2 * 60 * 60 // Two hours (7200 seconds)
```

Comment:
It is better to comment this like: hours * min * sec to not change the comment while changing value and it explains the magic numbers :)